astroquery/jplhorizons/core.py

```diff
@@ -594,6 +607,7 @@ def ephemerides_async(self, airmass_lessthan=99,
         if get_raw_response:
             self.return_raw = True
 
+
```

Comment:
One of the Travis jobs is failing because of the extra blank line:
`astroquery/jplhorizons/core.py:611:9: E303 too many blank lines (2)`

Could you amend this in the last commit and force push to this branch?

Reply:
Done!